src/legacy/core_plugins/kibana/public/discover/np_ready/angular/directives/field_name/field_name.tsx

```diff
@@ -19,8 +19,8 @@
 import React from 'react';
 import classNames from 'classnames';
 // @ts-ignore
-import { shortenDottedString } from '../../../../core_plugins/kibana/common/utils/shorten_dotted_string';
-import { FieldIcon } from '../../../../../../src/plugins/kibana_react/public';
+import { shortenDottedString } from '../../../../../../common/utils/shorten_dotted_string';
```

Comment:
It seems like `shorten_dotted_string` is also available from the data plugin

Reply:
Thanks for that hint!  Dound another usage that I could adapt. another @ts-ignore less, another brick in our old-kibana-wall. think we could now start to plan a museum of old code fragments. before everybody forgets.